custom-recipes/nlp-visualization-wordcloud/recipe.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
+from dataiku.customrecipe import get_recipe_resource
+from spacy_tokenizer import MultilingualTokenizer
+from wordcloud_visualizer import WordcloudVisualizer
+from plugin_config_loading import load_plugin_config_wordcloud
+
+# Load config
+params = load_plugin_config_wordcloud()
+resource_path = get_recipe_resource()
+output_folder = params["output_folder"]
+df = params["df"]
+
+# Load tokenizer
+tokenizer = MultilingualTokenizer()
+
+# Load wordcloud generator
+generator = WordcloudVisualizer(
+    tokenizer,
```

Comment:
Can you use explicit keyword assignment? like `a=a, b=truc`. This is more readable and avoid mistakes with argument misplacements.